packages/core/strapi/lib/commands/admin-create.js

```diff
@@ -95,14 +94,17 @@ async function createAdmin({ email, password, firstname, lastname }) {
   const appDir = process.cwd();
 
   const isTSProject = await tsUtils.isUsingTypeScript(appDir);
+  const compiledDirectoryPath = isTSProject
+    ? tsUtils.resolveConfigOptions(`${appDir}/tsconfig.json`).options.outDir
```

Comment:
Since we're reusing this logic a lot, could we consider moving it to the `typescript-utils`' utils lib?
Maybe something like "resolveOutDir" based on an appDir & an optional custom tsconfig.json filename?

Reply:
yeah that's good idea will add that to `typescript-utils`